cmd/ocitools/generate.go

```diff
@@ -463,3 +413,95 @@ func parseBindMount(s string) (string, string, string, error) {
 
 	return source, dest, options, nil
 }
+
+func addSeccomp(context *cli.Context, g *generate.Generator) error {
+	// Set the DefaultAction of seccomp
+	seccompDefault := context.String("seccomp-default")
+	err := g.SetDefaultSeccompAction(seccompDefault)
+	if err != nil {
+		return err
+	}
+
+	// Add the additional architectures permitted to be used for system calls
+	seccompArch := context.String("seccomp-arch")
+	architectureArgs := strings.Split(seccompArch, ",")
+	for _, arg := range architectureArgs {
+		err = g.SetSeccompArchitecture(arg)
```

Comment:
This works when you declare `err` before using `var err error` or `err := ....`. 
I pointed out because all the other for loops in this function seem to declare its own `err`.